src/platforms/hosted/cli.c

```diff
@@ -133,76 +134,102 @@ static void cl_help(char **argv)
 	PRINT_INFO(
 		"\n"
 		"Usage: %s [-h | -l | [-vBITMASK] [-d PATH | -P NUMBER | -s SERIAL | -c TYPE]\n"
-		"  [-n NUMBER] [-j] [-C] [-t | -T] [-e] [-p] [-R[h]] [-H] [-M STRING ...]\n"
-		"  [-E | -w | -V | -r] [-a ADDR] [-S number] [file]]\n"
+		"\t[-n NUMBER] [-j] [-C] [-t | -T] [-e] [-p] [-R[h]] [-H] [-M STRING ...]\n"
+		"\t[-f | -m] [-E | -w | -V | -r] [-a ADDR] [-S number] [file]]\n"
 		"\n"
-		"Single-shot and verbosity options [-h | -l | -vBITMASK]:\n"
```

Comment:
Thought we'd agreed to keep the section headers as they provide useful information on usage nearer the explanations?